CMakeLists.txt

```diff
@@ -1574,6 +1607,17 @@ if(APPLE)
   endif()
 endif()
 
+target_link_libraries(strawberry PUBLIC
+    strawberry_lib
+    lib-networkremote
+)
+
+target_link_libraries(strawberry PRIVATE Qt6::Protobuf)
```

Comment:
Protobuf should be moved to the existing target_link_libraries and use `Qt${QT_VERSION_MAJOR}::Protobuf` so it can still be used when Qt 7 is released.

Reply:
Will do